Example/Dev-Config.xcconfig

```diff
@@ -1,2 +1,2 @@
 #include "Dev-Keys.xcconfig"
-GCC_PREPROCESSOR_DEFINITIONS = $(inherited) GINI_CLIENT_ID=@\"$(GINI_CLIENT_ID)\" GINI_CLIENT_SECRET=@\"$(GINI_CLIENT_SECRET)\"
\ No newline at end of file
+GCC_PREPROCESSOR_DEFINITIONS = $(inherited) GINI_CLIENT_ID=@\"$(GINI_CLIENT_ID)\" GINI_CLIENT_SECRET=@\"$(GINI_CLIENT_SECRET)\"
```

Comment:
It might be my personal preferences, but before I commit, I always check all the changes and remove changes like that. They make it distracting when you're trying to review the code later because you don't only see the important changes but also loads of "noise".